examples/sparse/hypergraphatt.py

```diff
@@ -0,0 +1,126 @@
+"""
+Hypergraph Convolution and Hypergraph Attention
+(https://arxiv.org/pdf/1901.08150.pdf).
+"""
+import dgl
+import dgl.data
+import dgl.mock_sparse as dglsp
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torchmetrics.functional import accuracy
+import tqdm
+
+# Question:
+# The paper proposed two models, but HypergraphConv is essentially
+# what is in the HGNN paper (identical).  So should I rewrite HGNN
+# in the same fashion as this file?
+def hypergraph_laplacian(H):
+    ###########################################################
+    # (HIGHLIGHT) Compute the Laplacian with Sparse Matrix API
+    ###########################################################
+    d_V = H.sum(1)  # node degree
+    d_E = H.sum(0)  # edge degree
```

Comment:
Maybe just say `d_V = d_E = H.sum(0)`?

Reply:
I wrote the implementation as if the number of hyperedges does not equal the number of nodes so that people can adapt the code to other hypergraph datasets more easily.